sdk/eventhub/azure-eventhub/azure/eventhub/amqp/_amqp_message.py

```diff
@@ -34,15 +34,15 @@ class AmqpAnnotatedMessage(object):
     :keyword header: The amqp message header.
     :paramtype header: Optional[~azure.eventhub.amqp.AmqpMessageHeader]
     :keyword footer: The amqp message footer.
-    :paramtype footer: Optional[dict]
+    :paramtype footer: Optional[Dict]
```

Comment:
should be `dict or None`? as Laurent mentioned that "can add or None if the optionality matters, but we should clarify it in the text."

Reply:
Since this is a problem in a lot of places, not just this line, is it fine to leave it here and address it in the issue Laurent made?

Reply:
I agree that we should tackle one issue at a time.

As long as we ultimately update the sphinx docstring to be using `dict or None` instead of `Optional[Dict]`, I'm good.